src/polymer/elements/paper-toolbar-v1-to-v2.ts

```diff
@@ -0,0 +1,217 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import * as dom5 from 'dom5';
+import * as parse5 from 'parse5';
+import stripIndent = require('strip-indent');
+
+import {registry} from '../../registry';
+import {HtmlRule} from '../../html/rule';
+import {ParsedHtmlDocument, Severity, Warning} from 'polymer-analyzer';
+import {FixableWarning} from '../../warning';
```

Comment:
A couple changes needed from upstream work at master (after an `npm install`): 

- FixableWarning has been upstreamed into Warning
- `fix` has been added to the Warning constructor, and it's immutable now, so it has to be passed in that way.

Reply:
Fixed.